v1-1/learn/events.md

```diff
@@ -30,7 +30,7 @@ redirect_from:
 
 <!-- ## No upcoming events -->
```

Comment:
There are some broken code at the top and bottom of the file as shown below.
<img width="987" alt="Screenshot 2020-01-17 at 17 11 50" src="https://user-images.githubusercontent.com/11707273/72610212-d14cf200-394c-11ea-8e88-fe46638b4479.png">

<img width="989" alt="Screenshot 2020-01-17 at 17 12 04" src="https://user-images.githubusercontent.com/11707273/72610228-db6ef080-394c-11ea-8d5f-26eb70ae79a5.png">

We might need to fix those.